### ISSUE TITLE
fail a11y tasks with 0 urls found

### ISSUE THREAD

**Issue by drewbo** [cloud-gov/pages-cf-build-tasks#71]
Finding 0 urls is indicative of an upstream issue and we should fail the task

## Acceptance Criteria

- [ ] Determine next steps and acceptance criteria
